index.html

```diff
@@ -172,5 +172,29 @@ <h3 class="about-intro">LET'S CONNECT</h3>
     
         
     </section>
+    
+    <section class="contact-form">
+         <h2 class="contact-me">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it!
+            </p>
+            <form action="https://formspree.io/f/meqwrnzl" method="post">
+            <ul class="contact-list">
+                <li>
+                    <input type="text" id="name" placeholder="Full name" maxlength="50" required />
+                </li>
+                <li>
+                    <input type="email" id="email" placeholder="your email" required />
+                </li>
+                <li>
+                    <textarea rowa="6" cols="40" id="text" placeholder="Write your message here"
```

Comment:
- [ ] Although you have done a commendable job :+1:, I kindly request that you make a small addition to meet the project requirements. Specifically, it would be beneficial if you added the attribute 'required' to the text area. This will ensure that users cannot submit the form without filling out this field, and improve the overall usability and reliability of the form.